core/domain/exp_domain.py

```diff
@@ -655,6 +659,76 @@ def validate(self):
         self.outcome.validate()
 
 
+class Hint(object):
+    """Value object representing a hint."""
+    def __init__(self, hint_text, views):
+        self.hint_text = hint_text
```

Comment:
Make sure to use html_cleaner to scrub the hint text (see existing examples in this file).